org.eclipse.winery.common/pom.xml

```diff
@@ -104,12 +104,6 @@
             <version>${ch.qos.logback.logback-classic.version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
We need a logger for Tests in winery.common. even if there nicht be no Tests currently, I would say, we will miss addition of the logger when adding some